backend/src/routes/single_sign_on/index.js

```diff
@@ -0,0 +1,21 @@
+const controller = require("@controllers/single_sign_on/index.js");
+const validators = require("@validators/single_sign_on/index.js");
+
+const routes = [
+  {
+    method: "GET",
+    url: `/sso/users`,
+    schema: validators.getAll,
+    handler: controller.getUsers,
```

Comment:
i would have implicitly put this
`config: {
      role: "PMO-Admin-Edit-Capability",
    },`
or better yet fix this to use the new route config https://fastify.dev/docs/latest/Reference/Routes/#config